docs/api/browser-window.md

```diff
@@ -387,6 +387,13 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       Default is `true`.
     * `enableWebSQL` Boolean (optional) - Whether to enable the [WebSQL api](https://www.w3.org/TR/webdatabase/).
       Default is `true`.
+    * `V8CacheOptions` String (optional) - Enforces the v8 code caching policy
```

Comment:
```suggestion
    * `v8CacheOptions` String (optional) - Enforces the v8 code caching policy
```
To match the other properties (start with lowercase).

Reply:
Done, thanks!